### PR TITLE
chore: add files field into package.json

### PR DESCRIPTION
**Pull request by chouchouji**
# Question

I download `nrm` package globally from npm. It includes some files that I don't need actually, such as `.github`, `tests`, `.changeset` and so no. 

So I think we should only keep some necessary files to this cli to work. It is very helpful for users to reduce package size and save disk.

image

# Link

[https://docs.npmjs.com/cli/v10/configuring-npm/package-json#files](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#files)

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,12 @@
     "type": "git",
     "url": "git://github.com/Pana/nrm.git"
   },
+  "files": [
+    "actions.js",
+    "constants.js",
+    "helpers.js",
+    "registries.json"
```
Comment (iosh) [inline package.json:16]: Add  "registries.json",  "cli.js", "process.js" to files.

**Review by iosh — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (chouchouji) [reply package.json:16]: I had added `registries.json`. 

The  `cli.js` file will be added default. 

image

The `process.js` may be a test file?

image

Comment (iosh) [reply package.json:17]: `process.js` is used by helpers.js

Comment (chouchouji) [reply package.json:17]: ok, I see it. The comment mistakens me. I will update code.

**Comment by iosh** [comment]
Hi @chouchouji  if you want create new version you need add changeset file, Or we just merge this pr not release new version.

**Comment by chouchouji** [comment]
> Hi @chouchouji if you want create new version you need add changeset file, Or we just merge this pr not release new version.

I think we can test it by alpha version. But I don't know how to do it.

**Comment by chouchouji** [comment]
> npm pack

Thanks! `npm pack` is a good command. I test it and it works very well

image

**Comment by chouchouji** [comment]
Please tell me how to name it? 

image

**Comment by iosh** [comment]
If you want release new version 
```bash
npx changeset

# select patch

"Optimize package files Thanks @chouchouji"

```
Then git add the new file.

**Review by iosh — APPROVED** [review]
Great

**Comment by chouchouji** [comment]
> If you want release new version
> 
> ```shell
> npx changeset
> 
> # select patch
> 
> "Optimize package files Thanks @chouchouji"
> ```
> 
> Then git add the new file.

Thanks

**Pull request merged**